### PR TITLE
Rushikesh_Assignment_38_to_44

### PR DESCRIPTION
**Pull request by Rushikeshkapare**
Assignment-38
image
Assignment-39
image
Assignment-40
image
Assignment-41
image
Assignment-42
image
Assignment-43
image
Assignment-44
image

### CODE REVIEW

File: src/rushikesh/printPatternProgram/Assignment_44.java
Comment (tejasphase22) [inline src/rushikesh/printPatternProgram/Assignment_44.java:1]: Hello Rushikesh,
The logic is not correct here. Please run the program again and verify the output.

**Review by tejasphase22 — COMMENTED** [review]
Please recheck the assignment 44.

File: src/rushikesh/printPatternProgram/Assignment_44.java
Comment (Rushikeshkapare) [reply src/rushikesh/printPatternProgram/Assignment_44.java:1]: Hi tejas,
Please check now

Comment (tejasphase22) [reply src/rushikesh/printPatternProgram/Assignment_44.java:1]: Ya..it is correct now.

**Pull request merged**